src/web/Text.tsx

```diff
@@ -15,6 +15,14 @@ import RX = require('../common/Interfaces');
 import Styles from './Styles';
 import Types = require('../common/Types');
 
+if (typeof document !== 'undefined') {
```

Comment:
Please add a comment that explains what this does. It's sufficiently non-obvious that a comment is needed. :)